lib/msal-common/test/client/SilentFlowClient.spec.ts

```diff
@@ -306,17 +300,17 @@ describe("SilentFlowClient unit tests", () => {
             const config =
                 await ClientTestUtils.createTestClientConfiguration();
             const client = new SilentFlowClient(config, stubPerformanceClient);
-            const authResult = await client.acquireToken({
+            const authResult = await client.acquireCachedToken({
                 scopes: [],
                 account: testAccount,
                 authority: TEST_CONFIG.validAuthority,
                 correlationId: TEST_CONFIG.CORRELATION_ID,
                 forceRefresh: false,
             });
 
-            expect(authResult.account).toEqual(testAccount);
-            expect(authResult.idToken).toEqual(testIdToken.secret);
-            expect(authResult.accessToken).toEqual(
+            expect(authResult[0].account).toEqual(testAccount);
+            expect(authResult[0].idToken).toEqual(testIdToken.secret);
+            expect(authResult[0].accessToken).toEqual(
```

Comment:
I think you could add a test here to ensure that only one token was returned, and then set a const to equal the one token instead of using [0] three different times.

(and apply this feedback to the rest of this file, if you choose to implement it)

Reply:
This is because acquireCachedToken returns an array of type [AuthenticationResult, CacheOutcome]. The [0] isn't referring to the first of potentially many tokens.

Reply:
Right, thanks for reminding me of that.

But regarding the second half of my sentence above, look at line 170, pre-changes.

It's `const authResult: AuthenticationResult = response[0];`. Then `authResult` is used in `expect(...).toEqual(...)` instead of `response[0]` or `authResult[0]`.

I think we should keep this consistent throughout this file.